chart/templates/core-agents-deployment.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: core-agents
-  namespace: mayastor
+  namespace: {{.Values.namespace}}
```

Comment:
Should this be `{{ .Release.Namespace }}` ? As it stands this isn't actually picking up the -n provided to helm, it's using what you've defined in the yaml file?

Reply:
You're correct. Thanks, I've changed it.